src/components/action-popover/action-popover-interaction.stories.tsx

```diff
@@ -202,5 +201,57 @@ export const CloseOnEscPress: Story = {
     ),
   ],
 };
-
+CloseOnEscPress.parameters = { chromatic: { disableSnapshot: true } };
 CloseOnEscPress.storyName = "Close On ESC Press";
+
+export const SubmenuClick: Story = {
```

Comment:
I think the name of this story is misleading. There is no `Click` action to open the submenu. It is opened on hover.